meta-lmp-bsp/scripts/lib/wic/plugins/source/bootimg-sota-efi.py

```diff
@@ -128,7 +128,7 @@ def do_configure_partition(cls, part, source_params, creator, cr_workdir,
 
             # list of tuples (src_name, dst_name)
             deploy_files = []
-            for src_entry in re.findall(r'[\w;\-\./\*]+', boot_files):
+            for src_entry in re.findall(r'[\w;\-\.\+/\*]+', boot_files):
```

Comment:
@ricardosalveti does it make sense to upstream that change in the original recipe `./scripts/lib/wic/plugins/source/bootimg-efi.py`?

Reply:
I've sent one https://lists.openembedded.org/g/openembedded-core/message/210442

Reply:
Yup, thanks for sending upstream.